i18n/de/campaigns/tfa/the_depths_of_yoth.po

```diff
@@ -8,166 +8,174 @@ msgstr ""
 "The current depth level has no game effect on its own; however, some encounter card effects may change or become stronger depending on the current depth level."
-msgstr ""
+msgstr "In diesem Szenario gibt die Anzahl der Ressourcen unter „Aktuelle Tiefe“ auf der Szenarioübersichtskarte die aktuelle Tiefenstufe an. Wenn die Ermittler tiefer nach Yoth vordringen, indem sie Szene 1 abschließen, werden Ressourcen zur aktuellen Tiefenstufe hinzugefügt und das Spiel geht weiter, bis die Ermittler eine bestimmte Stufe erreicht haben oder besiegt werden. Tiefere Stufen von Yoth sind tödlicher, aber je tiefer die Stufe ist, die die Ermittler erreichen, desto besser wird die Auflösung des Szenarios für sie. Die aktuelle Tiefenstufe hat keinen eigenen Spieleffekt, aber einige Begegnungskarteneffekte können sich abhängig von der aktuellen Tiefenstufe verändern oder stärker werden."
 
 msgid "Pursuit"
-msgstr ""
+msgstr "Verfolgung"
 
 msgid ""
 "Some cards in this scenario reference an area called the “pursuit area.” The pursuit area is an out-of-play area next to the act and agenda deck which enemies can enter and leave via card effects. Generally speaking, enemies in play will move to the pursuit area whenever the investigators reach the next depth level. <b>Although the pursuit area is out of play, whenever an enemy moves to the pursuit area, it keeps all tokens and attachments.</b>\n"
 "Enemies in the pursuit area are considered to be “in pursuit.” Enemies that are in pursuit are considered to be out of play, and cannot be affected by player cards or investigator actions. However, enemies that are in pursuit can re-enter play by a variety of card abilities."
 msgstr ""
+"Einige Karten in diesem Szenario beziehen sich auf eine Zone namens „Die Verfolgungszone“. Die Verfolgungszone ist eine Zone außerhalb des Spiels, neben den Szenen- und Agendadecks, die Gegner durch Karteneffekte betreten und verlassen können. Im Allgemeinen werden Gegner im Spiel in die Verfolgungszone bewegt, sobald die Ermittler die nächste Tiefenstufe erreichen. <b>Obwohl sich die Verfolgungszone nicht im Spiel befindet, behalten Gegner, die in die Verfolgungszone bewegt werden, alle Marker und Verstärkungen.</b> \n"
+"Gegner in der Verfolgungszone gelten als „verfolgend.“ Verfolgende Gegner gelten als außerhalb des Spiels und können nicht von Spielerkarten oder Aktionen der Ermittler betroffen werden. Aber verfolgende Gegner können durch eine Reihe von Kartenfähigkeiten erneut ins Spiel kommen."
 
 msgid "the investigators fell into the depths."
-msgstr ""
+msgstr "Die Ermittler sind in die Tiefe gestürzt."
 
 msgid "Depth level when the scenario ended:"
-msgstr ""
+msgstr "Die Tiefenstufe am Ende des Szenarios:"
 
 msgid "#X# depth"
-msgstr ""
+msgstr "Tiefenstufe"
```

Comment:
So generally it's a good idea to keep the things in hashes as is, since the app can use that to interpolate a number.  I think in this situation its actually a hidden variable so it doesn't actually make a difference